.travis.yml

```diff
@@ -148,6 +164,14 @@ jobs:
 
     - stage: test
       name: 'macOS 10.12  [GOAL: deploy] [no functional tests]'
+      cache:
+        directories:
+          - $TRAVIS_BUILD_DIR/depends/built
+          - $TRAVIS_BUILD_DIR/depends/sdk-sources
+          - $TRAVIS_BUILD_DIR/ci/scratch/.ccache
+          - $TRAVIS_BUILD_DIR/releases/$HOST
+          - $HOME/Library/Caches/Homebrew
+          - /usr/local/Homebrew
```

Comment:
Isn't this cross compilation? I think this whole hunk can be discarded.

Reply:
Done.